lavalink/playermanager.py

```diff
@@ -1,7 +1,11 @@
+import logging
+
 from .node import Node
 from .models import BasePlayer
 from .exceptions import NodeException
 
+log = logging.getLogger('lavalink')
```

Comment:
`__name__`, to remain consistent with `__init__.py`, unless there's a specific reason for using `lavalink`?

Reply:
It was like that in `client.py`, want me to change it everywhere to that? I like that way since it's a bit more readable nothing major honestly.

Reply:
Yeah, good point. Could you rename any `getLogger(__name__)` to `getLogger('lavalink')` then please?